echopype/echodata/convention/1.0.yml

```diff
@@ -12,43 +12,84 @@ groups:
     name: Top-level
     description: contains metadata about the SONAR-netCDF4 file format.
     ep_group:
+    coords_ek60: []
+    coords_ek80: []
+    coords_azfp: []
+    coords_ad2cp: []
   environment:
     name: Environment
     description: contains information relevant to acoustic propagation through water.
     ep_group: Environment
+    coords_ek60: ['frequency', 'ping_time']
+    coords_ek80: ['ping_time']
+    coords_azfp: ['ping_time']
+    coords_ad2cp: ['ping_time', 'ping_time_burst', 'ping_time_average', 'ping_time_echosounder']
   platform:
     name: Platform
     description: contains information about the platform on which the sonar is installed.
     ep_group: Platform
+    coords_ek60: ['location_time', 'ping_time', 'frequency']
+    coords_ek80: ['mru_time', 'location_time']
+    coords_azfp: []
+    coords_ad2cp: ['ping_time', 'ping_time_burst', 'ping_time_average', 'ping_time_echosounder', 'beam',
+                   'range_sample_burst', 'range_sample_average', 'range_sample_echosounder']
   nmea:
     name: Platform/NMEA
     description: contains information specific to the NMEA protocol.
     ep_group: Platform/NMEA
+    coords_ek60: ['location_time']
+    coords_ek80: ['location_time']
+    coords_azfp: ['location_time']
+    coords_ad2cp: ['location_time']
   provenance:
     name: Provenance
     description: contains metadata about how the SONAR-netCDF4 version of the data were obtained.
     ep_group: Provenance
+    coords_ek60: []
+    coords_ek80: []
+    coords_azfp: []
+    coords_ad2cp: []
   sonar:
     name: Sonar
     description: contains specific metadata for the sonar system.
     ep_group: Sonar
+    coords_ek60: ['beam']
+    coords_ek80: ['beam', 'frequency']
+    coords_azfp: ['beam']
+    coords_ad2cp: []
```

Comment:
I can see in below the current `frequency` gets used as a coordinate for EK80. This should become `channel_id` per #566
https://github.com/OSOceanAcoustics/echopype/blob/1afc16aeb9b7ed67d39006ac9192f13b9d76e4b6/echopype/convert/set_groups_ek80.py#L92-L110

I am not sure why `beam` is listed here?

Reply:
> I can see in below the current frequency gets used as a coordinate for EK80. This should become `channel_id` per #566

Actually, I think we had settled on `channel`, not `channel_id`. But @leewujung please reply to my comment in #566 so we're 100% on the same page.

> I am not sure why `beam` is listed here?

I introduced this coordinate in the `Sonar` group in #574, to support our new "Beam Group" catalog variables. However, come to think of it, that name was a bad choice b/c it'll collide with the SONAR-netCDF4 `beam` dimension in the `Beam_groupX` groups. We should change it to `beam_group`.

Reply:
> I think we had settled on `channel`, not `channel_id`

Yes that is true, I agree with the choice of using `channel`. I replied there also and here again, just to make sure we are 100% certain. 

> We should change it to `beam_group`

Yes I think `beam_group` would better, in order to once and for all avoid confusion (hopefully!!).

Reply:
> Yes I think beam_group would better, in order to once and for all avoid confusion (hopefully!!).

I'll submit a separate PR